src/advanced_agent/config/loader.py

```
@@ -0,0 +1,314 @@
+    loader = ConfigLoader()
+    return loader.load_config(config_file=file_path)
+
+
+def create_default_config_file(file_path: str = "config/agent_config.yaml") -> AgentConfig:
+    """デフォルト設定ファイル作成（便利関数）"""
+    loader = ConfigLoader()
+    return loader.create_default_config(file_path)
+
+
+def validate_current_config() -> Dict[str, Any]:
+    """現在の設定検証（便利関数）"""
```

Ruff (0.12.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)